Documentation/kube-flannel-ds.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube-flannel-ds
+  namespace: kube-system
+  labels:
+    tier: node
+    app: flannel
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: flannel
+    spec:
+      hostNetwork: true
+      containers:
+      - name: kube-flannel
+        image: gcr.io/mikedanese-k8s/flannel:v9
```

Comment:
Might need to replace this with place-holder. Or a follow up PR once there is a tagged (or built from hash) release image. Just making note, but prob up to maintainers on preference.